nullaway/src/main/java/com/uber/nullaway/ErrorProneCLIFlagsConfig.java

```diff
@@ -56,7 +56,7 @@
   static final String FL_ACKNOWLEDGE_RESTRICTIVE =
       EP_FL_NAMESPACE + ":AcknowledgeRestrictiveAnnotations";
   static final String FL_CHECK_OPTIONAL_EMPTINESS = EP_FL_NAMESPACE + ":CheckOptionalEmptiness";
-  static final String FL_OPTIONAL_CLASS_PACKAGE = EP_FL_NAMESPACE + ":OptionalClassPath";
+  static final String FL_OPTIONAL_CLASS_PATHS = EP_FL_NAMESPACE + ":OptionalClassPaths";
```

Comment:
I still think this should be `CustomOptionalClasses`, otherwise I keep parsing it on my head as "Optional (class paths)" rather than "(Optional class) paths"

Reply:
Not that my alternative is perfect, but... mmm... maybe `CheckOptionalEmptinessCustomClasses`? A bit verbose, but at least ties the two options together.

Reply:
Changed it to `CheckOptionalEmptinessCustomClasses`, sounds better